app/templates/macros/page_macros.html

```diff
@@ -0,0 +1,49 @@
+{% macro render_inline_editor(editable_html_obj, current_user) %}
+    <div id="editor-{{ editable_html_obj.editor_name }}" contenteditable="false">
+        {{ editable_html_obj.value|safe }}
+    </div>
+
+    {% if current_user.is_admin() %}
+       <button class="ui button start-edit">
+        Edit
```

Comment:
Nit: should this be indented? you could probably put the whole thing on one line...
`<button class="ui button start-edit">Edit</button>